src/roiextractors/extraction_tools.py

```diff
@@ -26,6 +29,88 @@
 FloatType = float
 
 
+@dataclass()
```

Comment:
Do you need the `()` here? Usually @dataclass alone works, I thought

Reply:
No, this is not needed. I was thinking on making the dataclass [frozen](https://docs.python.org/3/library/dataclasses.html#frozen-instances) but I realize that it does not play that well with the `__post_init__`